app/code/Magento/CustomerGraphQl/Model/Resolver/GenerateCustomerToken.php

```diff
@@ -54,7 +68,11 @@ public function resolve(
 
         try {
             $token = $this->customerTokenService->createCustomerAccessToken($args['email'], $args['password']);
-            return ['token' => $token];
+            $tokenLifetime = $this->scopeConfig->getValue(self::TOKEN_LIFETIME_PATH_KEY);
```

Comment:
There is a helper method to get this value. Should we use it instead?
```php
Magento\Integration\Helper\Oauth\Data::getCustomerTokenLifetime()
```

Reply:
@codynguyen1 we could use the mentioned helper method. @tuanpt-0634 thank you for your suggestion